### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by Rabosa616** [opserver/Opserver#349]
Any idea to deploy into docker instead of an IIS?

**Comment by NickCraver** [comment]
Yes there is! The reason for there not being much noise is an active port to .NET Core 3.0. We have this up and working in Docker/K8s at Stack Overflow but still have a lot of issues to work through with config, secrets, etc.

If you want to try and give anything a spin, it's the `craver/aspnetcore` branch that I'm working in.

**Comment by NickCraver** [comment]
@Rabosa616 heads up: there's a `Dockerfile` up and working in #355. I need to put a lot of effort into docs in general though. Basically, mount the existing `/Config` directory to the image and the .NET Core version will pick it up.

**Comment by Rabosa616** [comment]
I was testing the repo and I can only build the image if I remove the copy nuget.config into the image due to if is there says something about ssl and no connection to the myget repos.
Does this happends to you also?

**Comment by NickCraver** [comment]
Nope not at all - do you have MyGet blocked by a firewall perhaps?
